app/decorators/case/base_decorator.rb

```diff
@@ -78,6 +78,10 @@ def trigger_case_marker
     end
   end
 
+  def highted_flag
```

Comment:
I think "highted" is not that clear - is the intention that this method can check if the case is 'high' priority? 

If so maybe just have `.high_priority?`